openapi3/schemas.py

```diff
@@ -88,7 +88,7 @@ def _parse_data(self):
         self.additionalProperties = self._get("additionalProperties", [bool, dict])
         self.description = self._get("description", str)
         self.format = self._get("format", str)
-        self.default = self._get("default", TYPE_LOOKUP.get(self.type, str))  # TODO - str as a default?
+        self.default = self._get("default", [list, dict, str, int, float, bool])  # TODO - str as a default?
```

Comment:
If we wanted to accept any type here, a wildcard would be more appropriate:

```suggestion
        self.default = self._get("default", "*")  # TODO - str as a default?
```